scripts/patch-debian-10.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# set the kernel version, like 5.0.21
+KV1=5
+KV2=0
+KV3=21
+# get the rt number, like rt10 on 4.x and rt16 on 5.x
+RT=16
+
+# I've tested 5.0.21 rt16 for Debian 10 and it works
+
+# use -c to continue existing downloads
+wget -c https://mirrors.edge.kernel.org/pub/linux/kernel/v5.x/linux-${KV1}.${KV2}.${KV3}.tar.xz
+wget -c https://mirrors.edge.kernel.org/pub/linux/kernel/projects/rt/${KV1}.${KV2}/patch-${KV1}.${KV2}.${KV3}-rt${RT}.patch.xz
+
+# use -k, keep the input files so they don't have to be redownloaded if you restart the script
+# use -f, overwrite existing in the case of restart
+unxz -kf linux-${KV1}.${KV2}.${KV3}.tar.xz
+unxz -kf patch-${KV1}.${KV2}.${KV3}-rt${RT}.patch.xz
+
+tar xf linux-${KV1}.${KV2}.${KV3}.tar
+cd linux-${KV1}.${KV2}.${KV3}
+
+patch -p1 < ../patch-${KV1}.${KV2}.${KV3}-rt${RT}.patch
+
+# this will apply all patches
+patch -p1 < ../*.patch
```

Comment:
@andrewhodel , you need to specify and provide the exact patches to be built into the kernel.
Though I don't see how this is going be implemented - Librealsense patches are different from kernel to kernel so you can't have a single version/ fit all approach.